src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -699,6 +769,48 @@ private void DrawOnline()
             EditorGUILayout.EndScrollView();
         }
 
+        private void DrawSelectedForInstallationHeader(GUIStyle headerStyle)
+        {
+            var rectangle = GUILayoutUtility.GetRect(GUIContent.none, headerStyle, GUILayout.Height(25f));
+
+            EditorGUI.LabelField(rectangle, string.Empty, headerStyle);
+
+            var arrow = showPackagesToInstall ? "\u2227" : "\u2228";
```

Comment:
I would have used some constants for the string so we don't have "magic numbers"